src/operators/courseItemOperations.ts

```diff
@@ -0,0 +1,64 @@
+import CourseItem, { CourseItemInterface } from '../models/courseItem';
+import { v4 as uuidv4 } from 'uuid';
+import { checkUserId } from './index';
+
+const checkCourseItemUser = checkUserId(CourseItem.get);
+
+const ERROR_RESPONSE = 'Course item not found.';
+
+export const create = async (courseItem: CourseItemInterface) => {
+	try {
+		const hashKey = uuidv4();
+		courseItem.id = hashKey;
+		// logs uuid of new courseItem for testing
+		console.log(courseItem.name + ' id: ' + courseItem.id);
+		await CourseItem.create(courseItem);
+	} catch (err) {
+		console.error(ERROR_RESPONSE);
+	}
```

Comment:
Remove the try/catch here. We need it to bubble up so we can catch it above. Same for all below. Catching as you did in `courseItem.ts` is good 😄